### PR TITLE
rabin2: add page

### PR DESCRIPTION
**Pull request by alufers**
Added page for the `rabin2` program.
https://radare.gitbooks.io/radare2book/content/rabin2/intro.html

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/rabin2.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# rabin2
+
+> Get information about binary files - symbols, sections, linked libraries, etc.
+> Comes bundled with radare2.
```
Comment (agnivade) [inline pages/common/rabin2.md]: Wrap radare2 within backticks (` `` `)

File: pages/common/rabin2.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# rabin2
+
+> Get information about binary files - symbols, sections, linked libraries, etc.
```
Comment (agnivade) [inline pages/common/rabin2.md]: What sort of binaries does it support ? ELF ? Mach-O ?

File: pages/common/rabin2.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Display symbols imported from libraries:
+
+`rabin2 -i {{path/to/binary}}`
+
+- Display strings contained in the binary:
+
+`rabin2 -z {{path/to/binary}}`
+
+- Output in JSON (works for every subcommand):
```
Comment (agnivade) [inline pages/common/rabin2.md]: Maybe it's better to say `Display the output in JSON:` and remove the -I option. You can combine any of the flags with other commands also (Please correct me if I am wrong). So I guess it is a bit redundant to say that.

Comment (alufers) [reply pages/common/rabin2.md]: You are right, but the `-j` option does not work on its own (it requires another flag), and I don't want to include examples that are broken.

Comment (agnivade) [reply pages/common/rabin2.md]: Oh I see. Alright then, just change the description. No need to remove `-I`

Comment (alufers) [reply pages/common/rabin2.md]: I have changed the description in the latest commit.

Comment (agnivade) [reply pages/common/rabin2.md]: I also wanted to remove the `(works for every subcommand)`. Because you can intermix the previous flags also.

Comment (alufers) [reply pages/common/rabin2.md]: Done.

Comment (agnivade) [reply pages/common/rabin2.md]: Thanks !

**Pull request merged**